src/Layout/PresentationMenu/PresentationSettings/PresentationSettings.tsx

```diff
@@ -149,12 +149,17 @@ export class PresentationSettingsMenu extends React.Component<PresentationSettin
         this.props.toggleUse2021Distribution(event.target.checked);
 
         const year = this.props.year;
-        const votes = this.props.votes.filter((vote) => vote.electionYear === year);
+        let votes = this.props.votes.filter((vote) => vote.electionYear === year);
```

Comment:
Maybe move this kind of logic into a function? It seems unclean to do it in here.

Reply:
Fixed.

Reply:
Why did this become a let?

Reply:
Because it may need to change, in case we merge the metrics from 19 down to 11 districts on line 157.